DEVELOPMENT.md

```diff
@@ -1,3 +1,59 @@
+# Getting Started
+
+For SoftServ development there are two repositories:
+
+- SoftServ's :: https://github.com/scientist-softserv/west-virginia-university
+- WVU's :: https://github.com/wvulibraries/hydra_acda_portal_public
+
+SoftServ's is a fork of WVU.
+
+In SoftServ there are two important branches: `main` and `softserv-dev`.  The `main` branch tracks to WVU's `main` branch; and likewise for `softserv-dev`.
+
+The SoftServ `main` has changes necessary for building your local docker instance.  Those changes should not be merged into `softserv-dev` nor into any of WVU's branches.
+
+## Procedure: Workflow
+
+- Clone SoftServ's repository
+- Checkout the SoftServ's `main` branch
+- Run `docker compose -f docker-compose.dev.yml build` to build the instance.
+- Once build, checkout `softserv-dev`.
+- Run `docker compose -f docker-compose.dev.yml up` to spin up the built instance.
+
+You need to start branches from `softserv-dev` and submit PRs to SoftServ's Github repository; the SoftServ team will review the changes and we then merge those changes into SoftServ's `softserv-dev` branch.
+
+At this point, we do local QA against SoftServ's `softserv-dev` branch.  When it passes internal QA, we can move the ticket. 
+
+We then need to send that code to WVU for review.  We also will submit PRs from SoftServ's `softserv-dev` branch to WVU's `softserv-dev` branch.  When we submit those, ping the developers at WVU to have them review and ultimately spin up a staging environment.
```

Comment:
Just curious and not trying to be combative, but why can't we just work off of WVU's repo?

Reply:
Not received as combative; the underlying challenge is multi-fold:

1. WVU's deploy ecosystem is notably different than ours.  So to create a Rancher environment would not provide much value as it would not reflect their ecosystem (and could be a diversion from the workflow).
2. The SoftServ `main` branch has docker customizations to help us spin up local instances in the least deviating manner possible.
3. WVU is not merging `softserv-dev` into their `main`; so there's a large accumulating drift.
4. WVU has a production application running this service that uses a different application logic, meaning that there are additional invisible layers.

And I think with that reality, it would be easier to accidentally pollute the WVU repository with our configurations.  I definitely agree that this divergence from our usual process is a risk (and adds some cognitive load).  So hopefully these instructions help provide guidance.